app/uk/gov/hmrc/apiplatformmicroservice/combinedapis/models/CombinedApi.scala

```diff
@@ -16,16 +16,22 @@
 
 package uk.gov.hmrc.apiplatformmicroservice.combinedapis.models
 
-import enumeratum.{Enum, EnumEntry, PlayJsonEnum}
+import scala.collection.immutable.ListSet
 
 import uk.gov.hmrc.apiplatform.modules.apis.domain.models._
 
-sealed trait ApiType extends EnumEntry
+sealed trait ApiType
 
-object ApiType extends Enum[ApiType] with PlayJsonEnum[ApiType] {
-  val values = findValues
+object ApiType {
+  val values: ListSet[ApiType] = ListSet(REST_API, XML_API)
```

Comment:
Can we make it a set unless it needs to be ordered.

Reply:
Wil do